### PR TITLE
EPMRPP-50925 || INFORMATION and WARNING statuses provided

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java [hunk 1]
```diff
@@ -48,8 +48,8 @@
 	private Date endTime;
 
 	@JsonProperty(value = "status")
-	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled" })
-	@ApiModelProperty(allowableValues = "PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED")
+	@In(allowedValues = { "passed", "failed", "stopped", "skipped", "interrupted", "cancelled", "info", "warn" })
```
Comment (HardNorth) [inline src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java:51]: New statuses does not conform the same pattern.  We have full status name for "interrupted" and "cancelled", but truncate "info" and "warn".

File: src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java [hunk 2]
```diff
@@ -48,8 +48,8 @@
...
+	@ApiModelProperty(allowableValues = "PASSED, FAILED, STOPPED, SKIPPED, RESETED, CANCELLED, INFO, WARN")
```
Comment (HardNorth) [inline src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java:52]: What was a reason of having status 'RESETED' and not having 'interrupted' here? I thought these two lines should be equal.

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java]: fixed

File: src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java:51]: It was designed to be "INFO&WARN" statuses both on the UI and back-end sides (technical requirements in JIRA task)

Comment (HardNorth) [reply src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java:51]: OK

File: src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java (hunk 2 above)
Comment (HardNorth) [reply src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java]: OK

**Pull request merged**